lib/pushmi_pullyu/aip/downloader.rb

```diff
@@ -1,215 +1,184 @@
+require 'fileutils'
+require 'ostruct'
+require 'rdf'
 require 'rdf/n3'
-require 'pushmi_pullyu/aip'
-require 'pushmi_pullyu/aip/solr_fetcher'
-require 'pushmi_pullyu/aip/fedora_fetcher'
 
 # Download all of the metadata/datastreams and associated data
 # related to an object
-
 class PushmiPullyu::AIP::Downloader
 
-  FILENAME_PREDICATE = 'info:fedora/fedora-system:def/model#downloadFilename'.freeze
+  class NoContentFilename < StandardError; end
 
   def initialize(noid)
     @noid = noid
-    @fedora_fetcher = PushmiPullyu::AIP::FedoraFetcher.new(@noid)
-    PushmiPullyu::AIP.validate(noid)
```

Comment:
the validations are all done before we get here. So we are validating this far more then we need to.